deploy/osd-pcap-collector/pcap-collector-scc.yaml

```
@@ -0,0 +1,20 @@
+kind: SecurityContextConstraints
+apiVersion: v1
+metadata:
+  name: pcap
+allowPrivilegedContainer: true 
+allowHostNetwork: true
+allowedCapabilities:
+- 'NET_ADMIN'
+- 'NET_RAW'
+runAsUser:
+  type: MustRunAsNonRoot
+seLinuxContext:
+  type: RunAsAny
```

We need to make sure this is also non-root.
```suggestion
  type: MustRunAsNonRoot
```

---

Ok. will test this in a cluster.

---

There isn't a MustRunAsNonRoot in the SELinux context. The only other option is to specify MustRunAs and manually specify the security context. Luis is trying out a couple of suggestions I have there.